File: project/backend/models/writing.js
'use strict';
module.exports = (sequelize, DataTypes) => {
  const Writing = sequelize.define('Writing', {
    writing_id: {
        allowNull: false,
        autoIncrement: true,
        primaryKey: true,
        type: DataTypes.INTEGER
    },
    title: {
      type: DataTypes.STRING
    },
    text: {
      type: DataTypes.TEXT
    },
    image: {
      type: DataTypes.TEXT
    },
    written_by: {
      allowNull: false,
      type: DataTypes.STRING
    },
    assignee: {
      type: DataTypes.STRING
    },
    lang_abbr:{
      type: DataTypes.STRING,
    }

  });
  Writing.associate = function(models) {
  //   models.Writing.belongsTo(models.User, {
  //     foreignKey: 'written_by',
  //     as: 'author',
  //     constraints: false
  // });
  };
  return Writing;
};

File: project/mobile/bulingo/app/src/main/java/com/bulingo/Database/Writing.java
package com.bulingo.Database;

import com.google.gson.annotations.SerializedName;

import retrofit2.SkipCallbackExecutor;

public class Writing {

    @SerializedName("writing_id")
    public int id;

    @SerializedName("title")
    public String title;

    @SerializedName("text")
   public String text;

    @SerializedName("written_by")
    public String writer;

    @SerializedName("assignee")
    public String assignee;
}
